<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QWidget" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MainPage{border:1px solid #92C8E8; margin: 0,0,0,0;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{border:1px solid #92C8E8; margin: 0px;background-color:white} QPushButton{border:0px;margin:0px;}
QLabel{border:0px;margin:0px;}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="styleSheet">
         <string notr="true">border:0px;margin:0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="minBtn">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="widget.qrc">
          <normaloff>:/resources/images/mininum</normaloff>:/resources/images/mininum</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="widget.qrc">
          <normaloff>:/resources/images/close</normaloff>:/resources/images/close</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <property name="styleSheet">
      <string notr="true">border:1px solid #92C8E8; background-color:#D6EFFB;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="bottomLayout">
        <item>
         <widget class="QWidget" name="bottomCenterWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>250</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>290</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget{border:0px;}
QLabel{color:#3F3A39;font-size:16px;font-family:楷体;}
QLineEdit{border:1px solid #CBD2D5 ;height:32px;border-radius:6px;background:white;padding-left:5px;}
QPushButton{border:0px;background:#F3A026;color:white;border-radius:6px;font-size:16px}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="avatarHLayout"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="singleTrainHLayout">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QPushButton" name="singleTrainBtn">
               <property name="minimumSize">
                <size>
                 <width>230</width>
                 <height>38</height>
                </size>
               </property>
               <property name="text">
                <string>单人训练</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="matchHLayout">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QPushButton" name="matchBtn">
               <property name="minimumSize">
                <size>
                 <width>230</width>
                 <height>38</height>
                </size>
               </property>
               <property name="text">
                <string>对战匹配</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="exitLoginHLayout">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>4</number>
             </property>
             <item>
              <widget class="QPushButton" name="exitLoginBtn">
               <property name="minimumSize">
                <size>
                 <width>230</width>
                 <height>38</height>
                </size>
               </property>
               <property name="text">
                <string>退出登录</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="widget.qrc"/>
 </resources>
 <connections/>
</ui>
